platform/ontology/api.go

```diff
@@ -44,100 +46,169 @@ func (p *Platform) GetTokenTxsByAddress(address string, token string) (blockatla
 		return blockatlas.TxPage{}, err
 	}
 	var txs []blockatlas.Tx
-	for _, srcTx := range txPage.Result.TxnList {
-		tx, ok := Normalize(&srcTx, token)
+	for _, txOntV1 := range txPage.Result.TxnList {
+		tx, ok := Normalize(txOntV1, token)
 		if !ok {
 			continue
 		}
-		txs = append(txs, tx)
+		txs = append(txs, *tx)
 	}
 
 	return txs, nil
 }
 
+func Normalize(txOntV1 Tx, assetName string) (tx *blockatlas.Tx, ok bool) {
+	if len(txOntV1.TransferList) < 1 {
+		return tx, false
+	}
+	transfer := txOntV1.TransferList[0]
+	fee := numbers.DecimalExp(txOntV1.Fee, ONGDecimals)
+	tx = &blockatlas.Tx{
+		ID:     txOntV1.TxnHash,
+		Coin:   coin.ONT,
+		Date:   txOntV1.TxnTime,
+		Block:  txOntV1.Height,
+		Status: getTransactionStatus(int(txOntV1.ConfirmFlag)),
+		Fee:    blockatlas.Amount(fee),
+	}
+
+	switch assetName {
+	case ONTAssetName:
+		normalizeONT(tx, transfer)
+	case ONGAssetName:
+		normalizeONG(tx, transfer)
+	default: // unsupported asset
+		return tx, false
+	}
+	return tx, true
+}
+
 func (p *Platform) CurrentBlockNumber() (int64, error) {
 	block, err := p.client.CurrentBlockNumber()
 	if err != nil {
 		logger.Error("CurrentBlockNumber", logger.Params{"platform": p.Coin().Symbol, "details": err.Error()})
 		return 0, err
 	}
 	var height int64
-	if block.Error != 0 {
-		err = errors.E("explorer error")
-	}
 	if len(block.Result) > 0 {
 		height = (int64)(block.Result[0].Height)
 	}
 	return height, nil
 }
```

Comment:
change the [Height variable](https://github.com/trustwallet/blockatlas/blob/0ddd73918ee26bfdfb4a96fdee4dd742b2a954b0/platform/ontology/model.go#L40) to int64 to avoid casting
```suggestion
	if len(block.Result) > 0 {
		return block.Result[0].Height
	}
	return 0, nil
```